falkordb-node/node-entrypoint.sh

```diff
@@ -267,11 +267,11 @@ wait_until_node_host_resolves() {
 
 get_master() {
   master_info=$(redis-cli -h $SENTINEL_HOST -p $SENTINEL_PORT --user $FALKORDB_USER -a $FALKORDB_PASSWORD $TLS_CONNECTION_STRING --no-auth-warning SENTINEL get-master-addr-by-name $MASTER_NAME)
-  redisRetVal=$?
+
   echo "Master Info: $master_info"
 
   # If RUN_SENTINEL is 1 and could not connect to sentinel, wait and try again
-  if [[ $RUN_SENTINEL -eq 1 && $redisRetVal -ne 0 ]]; then
+  if [[ $RUN_SENTINEL -eq 1 && -z $master_info ]]; then
```

Comment:
Here we're not necessarily going to get an empty response. We might get an error saying that there's no master for that group